spec/features/feature2_twats_order_spec.rb

```diff
@@ -0,0 +1,13 @@
+feature 'Messages are displayed in correct order' do
+  before(:each) { add_twats_to_db }
+  scenario 'A new message appears above old messages' do
+    visit('/')
+    login_to_twat
+    fill_in('new_message', with: 'this should be at the top')
+    click_button('Send')
+    within('div#messages') do 
+      # windows expect(page.text).to match(/this should be at the top\n\r\n\r\n\r\n29\/06\/1993 14:40\n\r\nAs a Maker/)
+      expect(page.text).to match(/this should be at the top\n29\/06\/1993 14:40\nAs a Maker/) # os
```

Comment:
your test should not have to know this OS. instead, why not use regexp to match what is displayed on screen?